qa/wildfly/build.gradle

```diff
@@ -63,8 +63,8 @@ elasticsearch_distributions {
   }
 }
 
-preProcessFixture {
-  dependsOn war, elasticsearch_distributions.docker
+tasks.named("preProcessFixture").configure {
+  dependsOn "war", elasticsearch_distributions.docker
```

Comment:
I see we wrap the task dependency in quotes here, but not later with `integTest`. In practice, does it matter? Obviously this config block is deferred until this task is needed, at which point we'll need all its dependencies as well, right? Essentially, there's no scenario in which this block will evaluate and we don't need to create the `war` task.

Reply:
indeed. there's only odd scenarios where this makes a difference I think. (e.g.) materialising this task without the need of having it in the task graph (by accident during configuration time). IMO ideally this materialisation would only happen by gradle itself when calculation the taskgraph